README.md

```diff
@@ -279,10 +83,18 @@ We have been working closely with the [`scanpy`](https://github.com/theislab/sca
 
 We are eager to explore integrations with other computational backends such as [`Seurat`](https://github.com/satijalab/seurat) or [`Bioconductor`](https://github.com/Bioconductor)
 
-## help and contact
+## communication
```

Comment:
Can we move this section up to after the quick start guide (and after the roadmap if we add that)? I think this is more important than developer communication (and overlaps slightly with that section).

Reply:
I agree.

Reply:
done. also renamed to "get in touch"